rocky/pyproject.toml

```diff
@@ -62,17 +57,15 @@ opentelemetry-instrumentation = "^0.41b0"
 opentelemetry-instrumentation-fastapi = "^0.41b0"
 granian = "^1.0.2"
 pydantic-settings = "^2.0.3"
+pyparsing = "^3.1.1"
```

Comment:
I also can't find a reference to this in rocky, so I think this should also not be top-level dependency?

Reply:
The problem is the way how to "include" Octopoes. Pyparsing is used by Octopoes' path module, which is used by Rocky for the reports. This is something I wanted to discuss with you soon.

Reply:
Ah you are right, I forgot about that...